### ISSUE TITLE
No explanation what to do with configuration file

### ISSUE THREAD

**Issue by konrazem** [enzymejs/enzyme#2546]
It is not a big issue, however, it is a bit frustrating that I can not find in the "Installation" section **what is the best way and how to implement** configured enzyme instance in my project:
```javascript
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });
```

If you can give an idea how to handle it it would be nice 😃

**Comment by ljharb** [comment]
That entirely depends on your test framework - and that framework's documentation usually would explain how you do a "setup" file for your test suite.

**Comment by konrazem** [comment]
Thanks @ljharb for the fast response. I understand. Even your comment to add to this section I think would be helpful with some examples. Like for create-react-app there is src/setupTests.js file that can do it. [There is even issue on stackoverflow ](https://stackoverflow.com/q/46627353/7838249)

**Comment by ljharb** [comment]
Yes, CRA uses jest, which has its own documentation for how to do a setup file.

A PR to improve the docs is welcome.

**Comment by jrachelr** [comment]
@ljharb I can work on this if you want to assign to me

**Comment by ljharb** [comment]
There’s no need for assignment; just make the PR and link it when you’re ready.
